testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/RPInitiatedLogoutTest.java

```diff
@@ -721,6 +725,51 @@ public void logoutWithBadClientId() {
     }
 
 
+    // Calling RP-Initiated Logout endpoint with POST request. This must be supported according to specification
+    @Test
+    public void logoutWithPostRequest() throws IOException {
+        try (RealmAttributeUpdater updater = new RealmAttributeUpdater(testRealm()).addSupportedLocale("cs").update()) {
+            OAuthClient.AccessTokenResponse tokenResponse = loginUser();
+
+            // Logout with POST request and automatic redirect after logout
+            String redirectUri = APP_REDIRECT_URI + "?logout";
+
+            String idTokenString = tokenResponse.getIdToken();
+            String sessionId = tokenResponse.getSessionState();
+
+            Map<String, String> postParams = new HashMap<>();
+            postParams.put(OIDCLoginProtocol.POST_LOGOUT_REDIRECT_URI_PARAM, redirectUri);
+            postParams.put(OIDCLoginProtocol.ID_TOKEN_HINT, idTokenString);
+            postParams.put(OAuth2Constants.STATE, "my-state");
+            URLUtils.sendPOSTRequestWithWebDriver(oauth.getLogoutUrl().build(), postParams);
+
+            events.expectLogout(tokenResponse.getSessionState()).detail(Details.REDIRECT_URI, redirectUri).assertEvent();
+            Assert.assertThat(false, is(isSessionActive(sessionId)));
```

Comment:
Please, always use `org.hamcrest.MatcherAssert.assertThat` instead. But, it's not a blocker for this PR, so you can keep it; it's just a nitpick.

Reply:
@mabartos Good point. I will fix that in the follow-up PR (as there is another PR related to logout I am going to send).